aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -3,7 +3,7 @@
 
 use crate::utils::*;
 use anyhow::Result;
-use aptos_types::on_chain_config::FeatureFlag as AFeatureFlag;
+use aptos_types::on_chain_config::{FeatureFlag as AFeatureFlag, Features as AFeatures};
```

Comment:
Let's in the future use names like `AptosFeatureFlag` instead of `AFeatureFlag`, the short adding (which I saw in the view functions as well), leads to easy mistakes and confusion later.